pym/bob/input.py

```diff
@@ -2463,6 +2489,9 @@ def __parse(self):
         self.__createSchemas()
 
         # user config(s)
+        self.__parseUserConfig("/etc/bobdefault.yaml")
```

Comment:
User defaults look quite useful to me. At least when I'm thinking about aliases. I'm frequently working on the same packages in different repositorys and so global aliases would be quite handy. What about a `~/.bobuser.yaml`? The concept of user defaults seems quite popular and understood to me.